cmd/nri-prometheus/config.go

```diff
@@ -94,6 +94,7 @@ func setViperDefaults(viper *viper.Viper) {
 	viper.SetDefault("require_scrape_enabled_label_for_nodes", true)
 	viper.SetDefault("scrape_timeout", 5*time.Second)
 	viper.SetDefault("scrape_duration", "30s")
+	viper.SetDefault("scrape_accept_header", "text/plain;version=0.0.4")
```

Comment:
Should this be an exposed config value? I think we should send the correct header for the things that the code supports. Users modifying this will almost surely lead to broken behavior.

Reply:
Talking offline we decided to change the header and support the value of the one in the prometheus server previous to the open-metric change 

https://github.com/prometheus/prometheus/commit/9c03e11c2cf2ad6c638567471faa5c0f6c11ba3d